be/src/olap/rowset/segment_v2/page_io.h

```diff
@@ -46,7 +46,7 @@ namespace segment_v2 {
 
 struct PageReadOptions {
     // block to read page
-    doris::io::FileReader* file_reader = nullptr;
+    io::FileReader* file_reader = nullptr;
```

Comment:
why not use shared pointer here?

Reply:
read_and_decompress_page is a synchronized method, this FileReader won't be released during reading page now.